docs/root/operations/cli.rst

```diff
@@ -20,6 +21,14 @@ following are the command line options that Envoy supports.
   <https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.text_format>`_
   formats respectively.
 
+.. option:: --config-json <json string>
```

Comment:
needs update to YAML, here and in release notes (probably also a master merge for release note fixing).